apps/web/src/components/Feed/Posts/PostHeader.tsx

```diff
@@ -0,0 +1,96 @@
+import { graphql, useFragment } from 'react-relay';
+import styled from 'styled-components';
+
+import InteractiveLink from '~/components/core/InteractiveLink/InteractiveLink';
+import { HStack, VStack } from '~/components/core/Spacer/Stack';
+import { BaseM, BaseS } from '~/components/core/Text/Text';
+import HoverCardOnUsername from '~/components/HoverCard/HoverCardOnUsername';
+import { ProfilePicture } from '~/components/ProfilePicture/ProfilePicture';
+import { PostHeaderFragment$key } from '~/generated/PostHeaderFragment.graphql';
+import { PostHeaderQueryFragment$key } from '~/generated/PostHeaderQueryFragment.graphql';
+import colors from '~/shared/theme/colors';
+import { getTimeSince } from '~/shared/utils/time';
+import { getCommunityUrlForToken } from '~/utils/getCommunityUrlForToken';
+
+import { StyledTime } from '../Events/EventStyles';
+import PostDropdown from './PostDropdown';
+
+type Props = {
+  postRef: PostHeaderFragment$key;
+  queryRef: PostHeaderQueryFragment$key;
+};
+
+export default function PostHeader({ postRef, queryRef }: Props) {
```

Comment:
the component `PostData`used to display all the sections within a Post, but I separated it into `PostHeader` and `PostNfts` so that it's easier to change the layout on desktop vs mobile

Reply:
Nice move!